oteps/4316-resource-provider.md

```diff
@@ -73,13 +176,15 @@ An EntityListener MUST provide the following operations:
 
 A `EntityProvider` MUST provide the following operations:
 
-* `Update Entity`
+* `Add or Update Entity`
+* `Replace Entity`
 * `Delete Entity`
 * `Get Resource`
```

Comment:
Most OTel APIs are write-only but this provides read access from the API. What reason would the instrumentation have to know the current resource?